content/blog/2019/01/2019-01-08-mpl-modular-pipeline-library.adoc

```diff
@@ -0,0 +1,386 @@
+---
+layout: post
+title: "MPL - Modular Pipeline Library"
+author: sparshev
+tags:
+- jenkinsfile
+- pipeline
+- sharedlibrary
+---
+
+NOTE: This is a guest post by Sergei Parshev from link:https://www.griddynamics.com/[Grid Dynamics], orignally posted on the
+link:https://blog.griddynamics.com/developing-a-modular-pipeline-library-to-improve-devops-collaboration/[Grid Dynamics Blog].
+
+Despite speeding up development with deployment automation, one of our clients
+was experiencing slow time-to-market due to a lack of collaboration in DevOps.
+While they had invested in DevOps, every production pipeline was set up
```

Comment:
Sorry, late to the train, can we clarify, maybe in a followup, what you mean by "collaboration in DevOps [...] invested in DevOps"?

Reply:
Hi @batmat , I really hope the terminology shouldn't be so hard to understand from context to make additional notes about "what does it mean" (otherwise we will come to "let's create dictionary somewhere to explain what I mean").
Of course everyone using different definitions of the "DevOps" term and it's quite wide, but Wikipedia are pretty clear about it: https://en.wikipedia.org/wiki/DevOps . Hope it will help.